OSDOCS-13191#Resizing recovery (#91198)
openshift/openshift-docs: storage/container_storage_interface/persistent-storage-csi-resizing.adoc

:_mod-docs-content-type: ASSEMBLY
[id="persistent-storage-csi-resizing"]
= CSI volume resizing
include::_attributes/common-attributes.adoc[]
:context: persistent-storage-csi-resizing

toc::[]

> Resizing persistent volumes allows you to easily resize an existing volume by editing a persistent volume claim (PVC). 

Reviewer: ```suggestion
You can resize an existing volume by editing a persistent volume claim (PVC). 
```
I revised this sentence to avoid anthropomorphic language ("allows you"). If my revision is inaccurate, feel free to ignore or change it.
